src/Maker/MakeResetPassword.php

```diff
@@ -0,0 +1,277 @@
+            ->addArgument('email-getter')
+            ->addArgument('password-setter')
+        ;
+
+        $io->section('- Email Templates -');
+        $emailText[] = 'Please answer the following questions that will be used to generate the email templates.';
+        $emailText[] = 'If you are unsure of what these answers should be, that\'s ok.';
+        $emailText[] = 'You can change these later after the templates have been generated.';
+        $io->text($emailText);
+
+        $input->setArgument('from-email-address', $io->ask(
+            'What email address will be used to send reset confirmations? I.e. admin@your-domain.com',
+            null,
+            [Validator::class, 'validateEmailAddress']
+        ));
+
+        $input->setArgument('from-email-name', $io->ask(
+            'What name will be associated with the email address used to send password reset confirmations? I.e. John Doe or Your Company, LLC.',
+            null,
+            [Validator::class, 'notBlank']
+        )
+        );
+
+        $interactiveSecurityHelper = new InteractiveSecurityHelper();
+
+        if (!$this->fileManager->fileExists($path = 'config/packages/security.yaml')) {
+            throw new RuntimeCommandException('The file "config/packages/security.yaml" does not exist. This command needs that file to accurately build the reset password form.');
+        }
+
+        $manipulator = new YamlSourceManipulator($this->fileManager->getFileContents($path));
+        $securityData = $manipulator->getData();
+        $providersData = $securityData['security']['providers'] ?? [];
+
+        $input->setArgument(
+            'user-class',
+            $userClass = $interactiveSecurityHelper->guessUserClass(
+                $io,
+                $providersData,
+                'Enter the User class that should be used with the "forgotten password" feature (e.g. <fg=yellow>App\\Entity\\User</>)'
+            )
+        );
+
+        $io->section('- Controller Template -');
+        $io->comment('<fg=yellow>A named route is required for redirection after a successful reset. Even routes that do not yet exist can be used here.</>');
+        $input->setArgument('controller-reset-success-redirect', $io->ask(
+            'What route should users be redirected to after their password has been successfully reset?',
+            'app_home',
+            [Validator::class, 'notBlank']
+        )
+        );
+
+        $io->text(sprintf('Implementing reset password for <info>%s</info>', $userClass));
+
+        $input->setArgument(
+            'email-property-name',
+            $interactiveSecurityHelper->guessEmailField($io, $userClass)
+        );
+        $input->setArgument(
+            'email-getter',
+            $interactiveSecurityHelper->guessEmailGetter($io, $userClass)
+        );
+        $input->setArgument(
+            'password-setter',
+            $interactiveSecurityHelper->guessPasswordSetter($io, $userClass)
+        );
```

Comment:
These 3 should be done immediately after guessing/asking the User class

Reply:
done